lib/services/elasticsearch.js

```diff
@@ -1024,6 +1025,34 @@ class ElasticSearch extends Service {
     return null;
   }
 
+  /**
+  * Sends an empty UpdateByQuery request to update the search index
+  *
+  * @param {String} index - Index name
+  * @param {String} collection - Collection name
+  * @returns {Promise.<Object>} {}
+  */
+  async updateSearchIndex(
+    index,
+    collection) {
+
+    let esRequest;
+    try {
```

Comment:
as it is, this `try..catch` statement is useless

did you mean to include `_client.update_by_query` in it?

Reply:
in deed